test/fixtures/rules/std_rule_cases/LT02-indent.yml

```diff
@@ -1919,3 +1919,25 @@ test_bigquery_merge_statement_values_clause:
   configs:
     core:
       dialect: bigquery
+
+test_fail_issue_4680:
+  # NOTE: It doesn't reindent the second clause, but the important
+  # thing is that we don't get an exception.
+  fail_str: |
+    SELECT col1
+    FROM table
+    WHERE
+      {% if true %}
+        col1 > 1
+      {% else %}
+        col1 > 0
+      {% endif %}
+  fix_str: |
+    SELECT col1
+    FROM table
+    WHERE
+        {% if true %}
+            col1 > 1
+        {% else %}
+        col1 > 0
+      {% endif %}
```

Comment:
these last 2 lines look to be in the wrong place? Aware this is less wrong than before, but do we want to fix?

Reply:
We do - but this is an example of only being able to fix what's rendered. The templater does explore that path so we can't fix it at the moment.

Barry was working on a pr to lint both paths in situations like this but it's not done yet.

Reply:
Ah makes sense - is that also valid for the endif?

Reply:
"valid" is probably inaccurate, more like "not covered" or "ignored", because it's not rendered yet.